ui/src/main/js/page/settings/standalone/SettingsEncryptionConfiguration.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import * as PropTypes from 'prop-types';
+import * as ConfigurationRequestBuilder from 'common/util/configurationRequestBuilder';
+import * as fieldModelUtilities from 'common/util/fieldModelUtilities';
+import PasswordInput from 'common/input/PasswordInput';
+import ConfigurationForm from 'page/channel/email/standalone/ConfigurationForm';
+import * as HttpErrorUtilities from 'common/util/httpErrorUtilities';
+import { SETTINGS_FIELD_KEYS, SETTINGS_INFO } from 'page/settings/SettingsModel';
+
+const SettingsEncryptionConfiguration = ({
+    csrfToken, errorHandler, readonly, displaySave
+}) => {
+    const encryptionRequestUrl = `${ConfigurationRequestBuilder.ENCRYPTION_API_URL}`;
+
+    const [settingsEncryptionConfig, setSettingsEncryptionConfig] = useState({});
+    const [errors, setErrors] = useState(HttpErrorUtilities.createEmptyErrorObject());
+
+    const [passwordFromApiExists, setPasswordFromApiExists] = useState(false);
+
+    const fetchData = async () => {
+        const response = await ConfigurationRequestBuilder.createReadRequest(encryptionRequestUrl, csrfToken);
+        const data = await response.json();
+
+        if (data) {
+            setSettingsEncryptionConfig(data);
+            setPasswordFromApiExists(true);
```

Comment:
If data is found and if the password field has content, mark that the password exists and delete that field from the data. Will need to do this for each password field

Reply:
The password field data from the server is the masked data correct?  So we empty the value and mark it as exists so the placeholder text is rendered correct?